scripts/install-chef-suse.sh

```diff
@@ -629,7 +629,7 @@ if [ -z "$CROWBAR_FROM_GIT" ]; then
       pattern=patterns-ses-admin
       pattern_short=ses_admin
     fi
-    if ! rpm -q $pattern &> /dev/null; then
+    if ! rpm -q --whatprovides $pattern &> /dev/null; then
         die "$pattern package is not installed; please install with \"zypper in -t pattern $pattern_short\" or \"zypper in $pattern\". Aborting."
```

Comment:
I was already aware that it was a pattern package, but my point was that a pattern package is still a pattern, at least semantically (IIRC there are 3 different types of pattern, so I guess there's no strict definition).  Otherwise the title of this PR would be "Check if a pattern provides the *package* we search for ..."  But anyway, it doesn't matter :)